indico/modules/events/registration/client/js/form/fields/TimetableSessions.jsx

```diff
@@ -295,3 +295,13 @@ export const timetableSessionsSettingsInitialData = {
   minimum: 0,
   maximum: 0,
 };
+
+export function SessionsSettingsFormValidator({minimum, maximum}) {
```

Comment:
```suggestion
export function sessionsSettingsFormValidator({minimum, maximum}) {
```

It's a function, not a class ;)